dotfiles.sh

```diff
@@ -39,8 +40,13 @@ if [ $version = yes ]; then
     echo "$versionstr"
     exit 0
 fi
+if [ $edit = yes ]; then
+    cd $HOME/.dotfiles
+    exec ${EDITOR:-vi} ${*:-.}
```

Comment:
Not all `$EDITOR`s might support editing a directory `.`. This then would fail here...

Reply:
Correct. The user is free to either provide a capable editor in $EDITOR or to explicitely name the file he or she wants to edit. If they refuse to provide the prereqs they won’t receive the benefits. That's something to be documented but I would not miss the fine feature of directory selection as most editors in use today (and preinstalled) can do it.